sickbeard/show_name_helpers.py

```diff
@@ -31,13 +31,15 @@
 from collections import namedtuple
 
 resultFilters = [
-    "sub(bed|ed|pack|s)",
     "(dir|sub|nfo)fix",
     "(?<!shomin.)sample",
     "(dvd)?extras",
     "dub(bed)?"
```

Comment:
Then add it too its as simple as changing the setting text, becouse as is you are ignoring scene releases without an option.
This is what was talked on irc @medariox